trussworks/react-uswds
# fix: Updates sign in stories to fix aria-controls issue of show password button

# Summary
Provides the correct id of password text input in the show password button.

## Related Issues or PRs
closed #2606 


## How To Test
1. Launch storybook and navigate to signin page in Page Templates 
![image](https://github.com/trussworks/react-uswds/assets/44632502/67d8a615-992b-4c7e-9729-abc8b315364d)
2. Click `open canvas in new tab` button to open this story in a new tab
![image](https://github.com/trussworks/react-uswds/assets/44632502/7cc601d0-8c76-4aca-b79f-594f53286a2a)

4. Run automated accessibility checker of choice on the page that opens to check for ARIA syntax issues
5.  Should not see an ARIA syntax issue (like below) related to the aria controls on the Show password link
![image](https://github.com/trussworks/react-uswds/assets/44632502/f5bc4260-7324-42c3-a62a-a27afa852607)





### Screenshots (optional)
![image](https://github.com/trussworks/react-uswds/assets/44632502/922c1b9d-e898-45b9-8bab-af92af8a70a1)
